python/ray/data/tests/test_huggingface.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import ray
+import datasets
+
+
+@pytest.fixture(scope="module")
+def ray_start_regular(request):  # pragma: no cover
+    try:
+        yield ray.init(num_cpus=16)
+    finally:
+        ray.shutdown()
```

Comment:
Any particular reason this fixture is needed rather than using one of the fixtures in `ray.tests.conftest`?